src/app/results/results.component.css

```diff
@@ -532,8 +546,20 @@ div.autocorrect{
   }
 }
 
+@media screen and (max-width: 375px) {
+  #search-options-field {
+    width: 92%;
+  }
+}
+
 @media screen and (max-width:330px) {
   #search-options li {
     padding: 0 2vw 12px;
   }
 }
+
+@media screen and (max-width: 320px) {
```

Comment:
@harshit98 It will. You are using max width right? If the max-width is 375px, than 320px is covered by it